src/components/pages/Navbar/Navbar.css

```diff
@@ -1,48 +1,38 @@
-.navBar{
-    display: flex;
-    justify-content: space-between;
-    background: #183b56 ;
-    z-index: 999;
-    width: 100%;
-    height: 10vh;
+.logoDiv {
```

Comment:
I really like the idea of placing all the css in another file instead of in line. We initially placed all these in line in Navbar.js, but it's much cleaner to place them in another file.